optuna/pruners/_hyperband.py

```diff
@@ -129,19 +129,25 @@ def objective(trial):
             A parameter for specifying reduction factor of promotable trials noted as
             :math:`\\eta` in the paper.
             See the details for :class:`~optuna.pruners.SuccessiveHalvingPruner`.
+        bootstrap_count:
+            Parameter specifying the number of trials required in a rung before any trial can be
+            promoted. Incompatible with max_resouce == 'auto'.
```

Comment:
super nit

```suggestion
            promoted. Incompatible with ``max_resouce`` is ``"auto"``.
```

Reply:
I admit that I don't quite understand this comment. I assume you are right, but I'm curious about the reasons.

- Are you refering to a style guide here (e.g. PEP 8)?
- Why use object identity (`is`) over object equality (`==`)?

Reply:
>     * Are you refering to a style guide here (e.g. PEP 8)?

No, if I understand what I do correctly. I said this from the perspective of how the documentation looks.  
I want every argument and variable to be double backquoted. With double backquotes, the value or expression like `max_resource / min_resource` rendered differently (orange characters) as in the capture below.
![image](https://user-images.githubusercontent.com/16191443/100956838-b97a5e80-355c-11eb-95d6-bf9ff70780b5.png)




>     * Why use object identity (`is`) over object equality (`==`)?

Object equality looks better :)

(I didn't mean to make this object identity by removing double quotes around "is" and I used "is" in order to make the sentence sounds natural when I read aloud.)

Reply:
Ah, I see that I became a bit confused here. Thanks for explaining. I was just asking, because I am always looking for coding styles to reduce the style variance in my own programming.